.gitignore

```diff
@@ -1,4 +1,4 @@
 .idea
 venv/
 pgdata/
-__pycache__/
\ No newline at end of file
+__pycache__/.ipynb_checkpoints
```

Comment:
What does this change exactly? This means anything in the `__pycache__` folder apart from `.ipynb_checkpoints` is added to git right? That's not what you want I guess? Or I forgot how `.gitignore` works.

Reply:
Yeah you're right. I'll change it to ignoring everything in pycache